composer.json

```diff
@@ -101,6 +105,9 @@
 
     "target-dir": "HWI/Bundle/OAuthBundle",
 
+    "minimum-stability": "dev",
```

Comment:
This is wrong, we don't support `dev` versions, we support only FOSUserBundle in `2.0@dev` nothing more, and with this we can get quite unexpected dependencies...